draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1835,6 +1846,12 @@ where the retrieval's URI is the request-uri, the retrieval's same-site status
 is computed for the HTTP request as defined in {{same-site-requests}}, and the
 retrieval's type is "HTTP".
 
+Note: While this spec requires that a single cookie-string be produced, a user
+agent may consider splitting large request headers within their {{RFC9110}}
```

Comment:
@reschke Not sure how you feel that I suggest implementers deviate from the http spec.

This problem isn't specific to cookies (even if cookies are much likely to encounter it) so I don't think this spec should directly suggest that the `cookie:` header should be split.

Reply:
Is this a thing any implementer does at the moment? It seems different in kind from the H2/H3 quirks discussed above... It's not clear to me that making this suggestion is helpful, and that the developer-facing admonition to reduce the number and size of cookies above would be more appropriate to focus on.

If implementers take this action today, it might be better to define how exactly such a split should be made, and likewise to define exactly how servers should handle multiple headers. If implementers don't take this action today, let's not tacitly approve of a choice to do so in the future?

Reply:
"While we require this, consider not following the requirement" feels very RFC 6919.

Reply:
Safari https://github.com/httpwg/http-extensions/issues/2541#issuecomment-1552101655 has been splitting the cookie header for a few years

Chrome does not and I don't believe Firefox does either (from my own testing)

Reply:
Yes, Safari started splitting Cookie in iOS 14 and macOS 11 (2020). It has become a load-bearing optimization since several internal websites won't load otherwise.

I was not aware that neither Chrome nor Firefox does it.

Reply:
After consideration I think this note should be removed.

Reply:
It might be worth a note to the opposite effect:

> Previous versions of this specification required that only one Cookie header fields be sent in requests.
> This is no longer a requirement. Some user agents split cookies across multiple header fields.
> See also {{Section 8.2.3 of ?RFC9113}} and {{Section 4.2.1 of ?RFC9114}}.

Reply:
I think that's a good way of phrasing it.